notebooks/py/assay_comparison.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+# coding: utf-8
+#
+# EPY: stripped_notebook: {"metadata": {"kernelspec": {"display_name": "Python 3", "language": "python", "name": "python3"}, "language_info": {"codemirror_mode": {"name": "ipython", "version": 3}, "file_extension": ".py", "mimetype": "text/x-python", "name": "python", "nbconvert_exporter": "python", "pygments_lexer": "ipython3", "version": "3.6.5"}}, "nbformat": 4, "nbformat_minor": 2}
+
+# EPY: START code
+# plotting
+# EPY: ESCAPE %matplotlib inline
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+sns.set_context('talk')
+sns.set_style('ticks')
+# EPY: END code
+
+# EPY: START code
+# munging
+import os
+# EPY: END code
+
+# EPY: START code
+# science
+from starfish import IntensityTable, Experiment, ImageStack
+from starfish.plot.intensity_table import compare_copy_number, spot_area_histogram, spot_distance_histogram, barcode_magnitude_histogram
+from starfish.plot import spots
+from starfish.types import Indices
+import numpy as np
+import pandas as pd
+
+from showit import image
+# EPY: END code
+
+# EPY: START markdown
+### Load IntensityTables
+# EPY: END markdown
+
+# EPY: START code
+# IntensityTable can't download from directories without list privileges
+
+data_root = "https://dmf0bdeheu4zf.cloudfront.net/assay_comparison/"
+iss_link = data_root + "iss.nc"
+merfish_link = data_root + "merfish.nc"
+dartfish_link = data_root + "dartfish.nc"
+
+tmp = os.environ['TMPDIR']
+iss_nc = tmp + "iss.nc"
+merfish_nc = tmp + "merfish.nc"
+dartfish_nc = tmp + "dartfish.nc"
+
+
+# EPY: ESCAPE !curl -o $iss_nc $iss_link
+# EPY: ESCAPE !curl -o $merfish_nc $merfish_link
+# EPY: ESCAPE !curl -o $dartfish_nc $dartfish_link
+
+iss = IntensityTable.load(iss_nc)
+merfish = IntensityTable.load(merfish_nc)
+dartfish = IntensityTable.load(dartfish_nc)
+# EPY: END code
+
+# EPY: START code
+datasets = [iss, merfish, dartfish]
+# EPY: END code
+
+# EPY: START markdown
+### Load Background Images
+# EPY: END markdown
+
+# EPY: START code
+# construct background images for each assay
+stack = Experiment.from_json('https://dmf0bdeheu4zf.cloudfront.net/20180821/DARTFISH/fov_001/experiment.json')
+
+dartfish_nuclei = stack.auxiliary_images['nuclei'].max_proj(Indices.CH, Indices.ROUND, Indices.Z)
+dartfish_link = data_root + "dartfish_dots_image.npy"
+dartfish_npy = tmp + "dartfish.npy"
```

Comment:
how was this dos_image created? is it the 'decoded_image' from the dartfish notebook?

Reply:
Max proj over hyb + channels.